api/views.py

```diff
@@ -14,11 +15,31 @@ class TeacherDetail(generics.RetrieveAPIView):
   queryset = Teacher.objects.all()
   serializer_class = TeacherSerializer
 
-  # def get(self, request, pk):
-  #   teacher = Teacher.objects.get(pk=pk)
 
-  #   serializer = TeacherSerializer(teacher)
-  #   return Response(serializer.data)
```

Comment:
Thanks for deleting this. I wasn't sure it it was needed for any other implementations